language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -41,9 +43,9 @@ abstractily specifiable. It's an implementation detail.
 
 -}
 
-type Plain f = f TyName Name ()
+type Plain f (uni :: * -> *) = f TyName Name uni ()
```

Comment:
IIRC there was a slow-going migration from `*` to `Type`.

Reply:
Done (in other places too), thanks.